metadata/ComputerVision__5__metadata.json

```diff
@@ -21,5 +21,6 @@
   "languageVersion": 3,
   "version": 5,
   "customVersion": "23.04.1",
-  "imagePath": "cv2304:23.4.5.4"
+  "imagePath": "cv2304:23.4.5.5",
+  "parametersFileJSON":  "{\"probes\": [{\"probe\": \"startup\",\"initialDelaySeconds\": \"0\",\"periodSeconds\": \"1\",\"timeoutSeconds\": \"1\",\"successThreshold\": \"1\",\"failureThreshold\": \"200\"}]}"
```

Comment:
I think its good to have some `initialDelaySeconds`. Maybe 120 minutes. It'll wait for 120 seconds before hitting startup probe

Reply:
Cloud is with GPU. Onprem is both GPU and CPU.